packages/libp2p/test/connection-manager/direct.spec.ts

```diff
@@ -402,6 +403,57 @@ describe('libp2p.dialer (direct, WebSockets)', () => {
     await libp2p.stop()
   })
 
+  it('should not run identify automatically after connecting', async () => {
+    libp2p = await createLibp2p({
+      peerId,
+      transports: [
+        webSockets({
+          filter: filters.all
+        })
+      ],
+      streamMuxers: [
+        yamux()
+      ],
+      connectionEncryption: [
+        plaintext()
+      ],
+      services: {
+        identify: identifyService({
+          protocolPrefix: 'ipfs',
+          agentVersion: AGENT_VERSION,
+          timeout: 60000,
+          maxInboundStreams: 1,
+          maxOutboundStreams: 1,
+          maxPushIncomingStreams: 1,
+          maxPushOutgoingStreams: 1,
+          maxObservedAddresses: 10,
+          maxIdentifyMessageSize: 8192,
+          runOnConnectionOpen: false
+        })
```

Comment:
This could be simpler to make it more obvious what the test is testing?

```suggestion
        identify: identifyService({
          runOnConnectionOpen: false
        })
```